src/libraries/System.Private.CoreLib/src/System/Int128.cs

```diff
@@ -1595,6 +1601,13 @@ private static bool TryConvertFromSaturating<TOther>(TOther value, out Int128 re
                          (actualValue == Half.NegativeInfinity) ? MinValue : (Int128)actualValue;
                 return true;
             }
+            else if (typeof(TOther) == typeof(BFloat16))
+            {
+                BFloat16 actualValue = (BFloat16)(object)value;
+                result = (actualValue >= new BFloat16(0x7F00)) ? MaxValue :
+                         (actualValue <= new BFloat16(0xFF00)) ? MinValue : (Int128)actualValue;
```

Comment:
nit: for `Half` we're using `== Half.PositiveInfinity` and `== Half.NegativeInfinity` for clarity.

We should do the equivalent here for `BFloat16`, either using an existing named constant or documenting what the boundary values used are.

Reply:
I meant conceptually the same. That is, we should use a well named member where it exists -or- document what value the bits represent.

`0x7F00` says nothing to the typical reader. Adding a comment that explains it means `(float)Int128.MaxValue` and corresponds roughly to `340282366920938463463374607431768211456` (adjusting for precision and the actual number stored) then means a lot